common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1235,19 +1220,48 @@
                     });
                 }
             },
-            handleTranscriptCookie: function() {
-                if ($.cookie('show_transcript') === null) {
-                    return;
+
+            /**
+            * This runs when the video block is first rendered and sets the initial visibility
+            * of the transcript panel based on the value of the 'show_transcript' cookie and/or
+            * the block's showCaptions setting.
+            */
+            setCaptionVisibility: function() {
```

Comment:
The code seems to often mix up "transcripts" and "captions", but since this is a new function about transcripts, can we call it `setTranscriptVisibility` to reduce that confusion?